src/components/org-users/views.tsx

```diff
@@ -627,7 +631,11 @@ export function InvitePage(props: IInvitePageProperties): ReactElement {
               name="email"
               type="text"
               defaultValue={props.values.email}
-              aria-describedby="email-hint email-error"
+              aria-describedby={
+                props.errors?.some(e => e.field === 'email')
+                  ? 'email-error'
+                  : ''
```

Comment:
Discussed offline, and having an empty string in the `aria-describedby` is legit and acceptable.

What we could do, is set it to `undefined`, which in the end _should_ prevent the attribute to be rendered out to the end HTML.

Alternatively, there is a potential of doing the following:

```typescript
const additionalAttributes = {};

additionalAttributes['aria-describedby'] = 'email-error';

return <input {...additionalAttributes} />;
```

Examples: https://jsfiddle.net/j0pkde3L/

This is non blocking thing and we can merge :)